src/lib/zendesk/zendesk.client.ts

```diff
@@ -0,0 +1,98 @@
+import logger from "../logger";
+
+const zendesk = require('node-zendesk')
+
+const zendeskClient = zendesk.createClient({
+  username: process.env.ZENDESK_USER,
+  token: process.env.ZENDESK_API_KEY,
+  remoteUri: process.env.ZENDESK_URL,
+  proxy: process.env.http_proxy
+})
+
+export async function updateTicketWithWorldpayGoLiveResponse(ticket: ZendeskTicket, requester: ZendeskUser, worldpayGoLiveUrl: string): Promise<boolean> {
+  logger.info(`Updating zendesk ticket ${ticket.id} with go live response...`)
+  try {
+    await zendeskClient.tickets.update(ticket.id,
+      {
+        ticket: {
+          // custom_fields: [{ id: 360012004639, value: "something"}], <-- This can be done in the future which would enable us to automatically close the ticket
+          comment: {
+            body: `
+Dear ${requester.name}
```

Comment:
Looks like `requester.name` only works for registered users. For others this may be email which won't look great. Maybe we can have something like `Hello` which we use in other macros to make it simple?